changelog/2022-09-02T12_48_27+02_00_static_files_tcl

```diff
@@ -0,0 +1,8 @@
+ADDED: The executable `static-files` in `clash-lib` can be used to write a Tcl
+script to a file that allows Vivado to interact with the metadata generated by
+Clash. The Tcl proc `clash::readHdl` will issue `read_vhdl`/`read_verilog` calls
+that add all the generated HDL files, and `clash::readXdc` will issue `read_xdc`
+commands for constraints on the top entity.
+
+You can write a copy of the Tcl script to a file by invoking
+    cabal run clash-lib:static-files -- --tcl-connector clashConnector.tcl
```

Comment:
```suggestion
    cabal run clash-lib:static-files -- --tcl-connector=clashConnector.tcl
```
?

Reply:
Apart from my personal preference to omit the `=`, the `=` also prevents bash from expanding a `~` leading the path, giving our users grief with errors like "Directory does not exist: ~". So, no, I'd rather keep it like this.

Reply:
Well what I meant was: the functions docopt uses `--tcl-connector=....`. So can we actually use this form?

Reply:
Yes, it is fine. I think you have to use the equals sign in the usage message, though.